extensions/expanded-registers/README.md

```diff
@@ -36,5 +36,4 @@ When used with an instruction that uses an `A` register, `REX.A` provides a 4th
 When used with an instruction that uses a `B` register (including a `SIB.B` if [MO1](../memory-operands-1/README.md) or [MO2](../memory-operands-2/README.md) is available), `REX.B` provides a 4th bit for the specifier in the same way.
 When used with an instruction that uses an `SIB.X` register, `REX.X` provides a 4th bit for the specifier in the same way.
 
-`REX.Q` is reserved.
-
+Unused bits _should_ be set to 0 and are it undefined behavior for them to be set when an instruction does not use them unless specified otherwise.
```

Comment:
Did you mean?
```suggestion
Unused bits _should_ be set to 0 and are undefined behavior for them to be set when an instruction does not use them unless specified otherwise.
````

Reply:
I did make a typo there, but not that exact typo.